src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(arg) {
+    if (arg === undefined) {
+      const result = sum;
+
+      sum = 0;
+
```

Comment:
Consider adding a blank line here to separate the logic of resetting the sum from the return statement. This can enhance readability.